AirSkyBoat/AirSkyBoat
# Bugfix/to cure a cough page 3 fix

**_I affirm:_**

## What does this pull request do?



This PR addresses an issue where "Page 3" of the Diary in Southern San d'Oria does not display even if the player has begun the quest "To Cure a Cough". Per the wiki page the quest does not get officially accepted and doesn't display in the quest log until it's complete. The current check on the quest expects the quest to be accepted but there is a variable "toCureaCough" that should be checked instead.

The rest of the quest was able to be progressed and completed successfully during my testing.

Source: https://ffxiclopedia.fandom.com/wiki/To_Cure_a_Cough
Fixes: https://github.com/AirSkyBoat/AirSkyBoat/issues/2406

## Steps to test these changes



1. Complete prerequisite quests "The Trader in the Forest" (Log Id 0, Quest Id 7) and "The Medicine Woman" (Log Id 0, Quest Id 30). 
2. Complete additional, unrelated quest "A Squire's Test II" (log Id 0, Quest Id 19).
3. Talk to Nenne in Southern San d'Oria.
4. Use the Diary and confirm page 3 displays. (This is the only step this PR addresses)
5. Talk to Amaura in Southern San d'Oria and confirm she requests Thyme Moss.
6. Go to Davoi and click on the ??? object at F-9
7. Confirm you receive the Thyme Moss key item.
8. Return to Amaura and talk to her to receive Cough Medicine.
9. Return to Nenne to receive a Scroll of Treasure key item.
10. Confirm the quest displays on your completed quests list.
